src/sonic-yang-models/doc/Configuration.md

```diff
@@ -740,8 +740,9 @@ This kind of profiles will be handled by buffer manager and won't be applied to
 
 ### Chassis Module
 
-CHASSIS_MODULE table holds the list and configuration of linecard and fabric modules in a SONiC chassis.
-It currently allows user to administratively bring down a line-card or fabric-card
+CHASSIS_MODULE table holds the list and configuration of linecard, DPU modules and fabric modules in a SONiC chassis.
+It currently allows user to administratively bring down a line-card or DPU or fabric-card
+A smartswitch chassis will not have LINE-CARD and FABRIC-CARD.  It will only have DPUs as shown

```

Comment:
Nits: use the code formatted `LINE-CARD` instead of plain LINE-CARD so readers know you are referring to a json field. Same for the others.